### DISCUSSION TITLE
Plugin Drag-n-Drop not working under Plasma Wayland

### DISCUSSION THREAD

**Discussion by jSQrD-dev** [audacious-media-player/audacious#72] [General]
```
Operating System: EndeavourOS 
KDE Plasma Version: 5.27.6
KDE Frameworks Version: 5.108.0
Qt Version: 5.15.10
Kernel Version: 6.4.6-zen1-1-zen (64-bit)
Graphics Platform: Wayland
Processors: 16 × Intel® Core™ i7-10875H CPU @ 2.30GHz
Memory: 15.4 GiB of RAM
Graphics Processor: Mesa Intel® UHD Graphics
Manufacturer: Dell Inc.
Product Name: XPS 17 9700
```

Drag-n-Drop of Plugins does not seem to work under Wayland and placing plugins in X11 and switching back to Wayland undocks the plugin.

I have a feeling that this might be because applications can not listen to actions taken in other apps in Wayland.

**Comment by jlindgren90** [comment]
It is a Qt bug/limitation, supposedly fixed in Qt 6-point-something. I would suggest using XWayland to run Audacious for now.
